test/tax-lot/tax-lot.repository.mock.ts

```diff
@@ -2,22 +2,51 @@ import { generateMock } from "@anatine/zod-mock";
 import {
   findByBblRepoSchema,
   findByBblSpatialRepoSchema,
+  findZoningDistrictByTaxLotBblRepoSchema,
+  checkTaxLotByBblRepoSchema,
 } from "src/tax-lot/tax-lot.repository.schema";
 
 export class TaxLotRepositoryMock {
-  findByBblMocks = Array.from(Array(10), () =>
-    generateMock(findByBblRepoSchema, { seed: 1 }),
+  NUMBER_OF_MOCKS = 1;
```

Comment:
Because this is held within a class, I think camelCase would be fine- I've only seen truly global variables use SCREAMING_SNAKE_CASE. But, I'm not hung up over it.

Reply:
Updated to camel case 👍